cmd/create/operatorroles/cmd.go

```diff
@@ -377,3 +386,30 @@ func getVersionMinor(cluster *cmv1.Cluster) string {
 	segments := version.Segments()
 	return fmt.Sprintf("%d.%d", segments[0], segments[1])
 }
+
+func validateOperatorRoles(awsClient aws.Client, cluster *cmv1.Cluster) ([]string, error) {
+	var missingRoles []string
+
+	operatorIAMRoles := cluster.AWS().STS().OperatorIAMRoles()
+
+	if len(operatorIAMRoles) == 0 {
+		return missingRoles, fmt.Errorf("No Operator IAM roles found for cluster %s", cluster.Name())
+	}
+
+	for _, operatorIAMRole := range operatorIAMRoles {
+		roleARN := operatorIAMRole.RoleARN()
```

Comment:
@vkareh `v1.OperatorIAMRole.Name()` doesn't return the actual role name it looks like it returns an internal name? So I had to split the ARN.

Reply:
Name is the operator name (there's also `.Namespace()`)

Reply:
Maybe I should have named this attribute `cluster.AWS().STS().CredentialRequests()` or something, but I started with the `OperatorIAMRole` and it stuck even after adding the name/namespace stuff :/

Reply:
Naming is hard :smile: I was just curious, thanks!